go/tasks/plugins/k8s/kfoperators/common/common_operator.go

```diff
@@ -104,8 +105,11 @@ func GetMPIPhaseInfo(currentCondition commonOp.JobCondition, occurredAt time.Tim
 }
 
 // GetLogs will return the logs for kubeflow job
-func GetLogs(taskType string, name string, namespace string, hasMaster bool,
+func GetLogs(taskType string, objectMeta meta_v1.ObjectMeta, hasMaster bool,
```

Comment:
By passing in `objectMeta` as opposed to individually passing in the name, namespace and in the future also the creation timestamp, we can retrieve the creation timestamp only a single time here instead of doing so in `pytorch.go`, `tensorflow.go`, as well as `mpi.go`.